administrator/language/en-GB/plg_schemaorg_recipe.ini

```
@@ -0,0 +1,38 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SCHEMAORG_RECIPE="Schema.org - Recipe"
+PLG_SCHEMAORG_RECIPE_FIELD_AUTHOR_LABEL="Author"
+PLG_SCHEMAORG_RECIPE_FIELD_CALORIES_LABEL="Calories"
+PLG_SCHEMAORG_RECIPE_FIELD_CARBOHYDRATE_LABEL="Carbohydrate"
+PLG_SCHEMAORG_RECIPE_FIELD_CATEGORY_LABEL="Category"
+PLG_SCHEMAORG_RECIPE_FIELD_CHOLESTROL_CONTENT_LABEL="Cholestrol"
+PLG_SCHEMAORG_RECIPE_FIELD_COOKING_MEHTOD_LABEL="Cooking Method"
+PLG_SCHEMAORG_RECIPE_FIELD_COOK_TIME_LABEL="Cook Time"
+PLG_SCHEMAORG_RECIPE_FIELD_CUISINE_LABEL="Cuisine"
+PLG_SCHEMAORG_RECIPE_FIELD_DATE_PUBLISHED_LABEL="Date Published"
+PLG_SCHEMAORG_RECIPE_FIELD_DESCRIPTION_LABEL="Description"
+PLG_SCHEMAORG_RECIPE_FIELD_FAT_LABEL="Fat"
+PLG_SCHEMAORG_RECIPE_FIELD_FIBER_LABEL="Fiber"

```

```suggestion
PLG_SCHEMAORG_RECIPE_FIELD_FIBER_LABEL="Fibre"

```

This is the en-GB spelling